.travis.yml

```diff
@@ -5,13 +5,49 @@ node_js:
   - '8'
   - '6'
 install:
+  - 'if [ -n "${JEST}" ]; then yarn add --dev "jest@${JEST}"; fi'
   - 'if [ -n "${ESLINT}" ]; then yarn add "eslint@${ESLINT}"; fi'
   - 'yarn'
 env:
-  - 'ESLINT=6'
-  - 'ESLINT=5'
-  - 'ESLINT=4'
+  - 'ESLINT=4 JEST=21'
+  - 'ESLINT=5 JEST=21'
+  - 'ESLINT=6 JEST=21'
+  - 'ESLINT=4 JEST=22'
+  - 'ESLINT=5 JEST=22'
+  - 'ESLINT=6 JEST=22'
+  - 'ESLINT=4 JEST=23'
+  - 'ESLINT=5 JEST=23'
+  - 'ESLINT=6 JEST=23'
+  - 'ESLINT=4 JEST=24'
+  - 'ESLINT=5 JEST=24'
+  - 'ESLINT=6 JEST=24'
+  - 'ESLINT=4 JEST=25'
+  - 'ESLINT=5 JEST=25'
+  - 'ESLINT=6 JEST=25'
 matrix:
   exclude:
+    # exclude eslint 6 on node 6
     - node_js: '6'
-      env: 'ESLINT=6'
+      env: 'ESLINT=6 JEST=21'
+    - node_js: '6'
+      env: 'ESLINT=6 JEST=22'
+    - node_js: '6'
+      env: 'ESLINT=6 JEST=23'
+    - node_js: '6'
+      env: 'ESLINT=6 JEST=24'
+    - node_js: '6'
+      env: 'ESLINT=6 JEST=25'
+    # exclude jest 25 on node 6
+    - node_js: '6'
+      env: 'ESLINT=4 JEST=25'
+    - node_js: '6'
+      env: 'ESLINT=5 JEST=25'
+    - node_js: '6'
+      env: 'ESLINT=6 JEST=25'
+    # exclude jest 25 on node 8
```

Comment:
why? Jest 25 still works with node 8

Reply:
This is because jest-snapshot v25 requires a minimum of semver 7.1.1, which requires node 10:

![image](https://user-images.githubusercontent.com/836644/72957252-29766080-3d60-11ea-8308-a276de9de1df.png)

Reply:
Then that’s probably a bug in jest-snapshot, which should still support node 8 whole jest does?

Reply:
Yeah I would think so, but not sure if that should block getting a version of this landed that supports jest 25. What do you think? Happy to file an issue.

Reply:
Agree it shouldn’t block for now; filing a jest issue would be great too.

Reply:
Issue filed here: https://github.com/facebook/jest/issues/9450

Reply:
merged it, but I'm not sure when we'll get a patch release out. Omitting it for now in the testing matrix here makes sense